python/arcticdb/version_store/_normalization.py

```diff
@@ -1040,7 +1054,16 @@ def _custom_pack(self, obj):
     def _ext_hook(self, code, data):
         if code == MsgPackSerialization.PD_TIMESTAMP:
             data = MsgPackNormalizer._nested_msgpack_unpackb(data)
-            return pd.Timestamp(data[0], tz=data[1]) if data[1] is not None else pd.Timestamp(data[0])
+            if len(data) == 2:
```

Comment:
Well worth a comment here to explain the old 2-element data format and that this is for backwards compat, as the need for the 2 element case is impossible to understand from the code itself.